walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -1,25 +1,115 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+@media screen and (min-width: 600px)
```

Comment:
wrapping all the CSS rules in a media query seems odd to me. This means that accessing from a phone will load no css rules.
🠪 1. is the goal to only load CSS rules for tablet/desktop because mobile styling is handled in a separate PR?
🠪 2. OR, is mobile styling not covered at all in your starter projects?

Reply:
Branch RCAVelez-redesignIndex has removed media query.